RELEASE_PROCESS.md

```diff
@@ -0,0 +1,36 @@
+# Building a Compose release
+
+## Building binaries
+
+`script/build-linux` will build the Linux binary inside a Docker container:
```

Comment:
So, future tense is unnecessary.  Style guidance is:
![image](https://cloud.githubusercontent.com/assets/1347209/8460001/e85f766a-1fd5-11e5-8c65-f53dd237e129.png)